tests/test_cell_identity.py

```diff
@@ -0,0 +1,27 @@
+from itertools import pairwise
+
+from telcell.cell_identity import CellIdentity, RADIO_GSM, RADIO_UMTS, RADIO_LTE, RADIO_NR
+
+CELL_IDENTITIES = [
+    ("204-1", CellIdentity.create(mcc=204, mnc=1)),
+    ("GSM/204-1", CellIdentity.create(radio=RADIO_GSM, mcc=204, mnc=1)),
+    ("204-2", CellIdentity.create(mcc=204, mnc=2)),
+    ("204-1-2000", CellIdentity.create(mcc=204, mnc=1, eci=2000)),
+    ("204-1-2-2000", CellIdentity.create(mcc=204, mnc=1, lac=2, ci=2000)),
+    ("GSM/204-1-2-2000", CellIdentity.create(radio=RADIO_GSM, mcc=204, mnc=1, lac=2, ci=2000)),
+    ("UMTS/204-1-2-2000", CellIdentity.create(radio=RADIO_UMTS, mcc=204, mnc=1, lac=2, ci=2000)),
+    ("LTE/204-1-2000", CellIdentity.create(radio=RADIO_LTE, mcc=204, mnc=1, eci=2000)),
+    ("NR/204-1-2000", CellIdentity.create(radio=RADIO_NR, mcc=204, mnc=1, eci=2000)),
+    ("GSM/204-1-?-?", CellIdentity.create(radio=RADIO_GSM, mcc=204, mnc=1)),
```

Comment:
Would the first, second and last entries effectively encode the same cellid? If so: would that be worth testing for?